grammar.js

```diff
@@ -373,6 +378,39 @@ module.exports = grammar({
       seq(repeat($._annotation), '[', ']')
     )),
 
+    switch_construct: $ => seq(
```

Comment:
Can you rename this to `switch_expression`? I know it's not always a switch expression, but I think the `construct` terminology is a bit inconsistent with other rules. If we want to be more precise, we could alias it to `switch_statement` within the `_statement` rule. I'd be ok either with or without the alias.